src/nginxconfig/templates/domain_sections/logging.vue

```diff
@@ -26,35 +26,123 @@ THE SOFTWARE.
 
 <template>
     <div>
-        <div class="field is-horizontal">
-            <div class="field-label">
+        <div class="field is-horizontal is-aligned-top">
+            <div class="field-label has-small-margin-top">
                 <label class="label">access_log {{ $t('templates.domainSections.logging.byDomain') }}</label>
             </div>
             <div class="field-body">
                 <div class="field">
-                    <div :class="`control${accessLogChanged ? ' is-changed' : ''}`">
+                    <div :class="`control${accessLogEnabledChanged ? ' is-changed' : ''}`">
                         <div class="checkbox">
-                            <PrettyCheck v-model="accessLog" class="p-default p-curve p-fill p-icon">
+                            <PrettyCheck v-model="accessLogEnabled" class="p-default p-curve p-fill p-icon">
                                 {{ $t('templates.domainSections.logging.enableForThisDomain') }}
                             </PrettyCheck>
                         </div>
                     </div>
+                    <div v-if="$props.data.accessLogEnabled.computed" :class="`control field is-horizontal is-expanded${accessLogPathChanged ? ' is-changed' : ''}`">
+                        <input
+                            v-model="accessLogPath"
+                            class="input"
+                            type="text"
+                            :placeholder="$props.data.accessLogPath.default"
```

Comment:
When empty, the placeholder shown doesn't seem to match what actually gets used? The version in the config injects the domain, but here it doesn't

Reply:
I'm a bit confused here - didn't we discuss that we can't suggest a different name per domain because we can't tell when we are migrating from an old configuration and when not? And that we should use the global filenames after all, where the user might be able to customize it if they want.

Reply:
now I get it, thanks.